contracts/chain-adapters/Arbitrum_CustomGasToken_Adapter.sol

```diff
@@ -116,6 +122,7 @@ contract Arbitrum_CustomGasToken_Adapter is AdapterInterface, CircleCCTPAdapter
         L2_MAX_SUBMISSION_COST = _l2MaxSubmissionCost;
         L2_GAS_PRICE = _l2GasPrice;
         CUSTOM_GAS_TOKEN_FUNDER = _customGasTokenFunder;
+        NATIVE_TOKEN_DECIMALS = _nativeTokenDecimals;
```

Comment:
Is it easier to set this by calling `CUSTOM_GAS_TOKEN.decimals()` instead? It'd save a constructor argument so it's one less thing we can get wrong.

Reply:
The ERC20 standard actually [doesn't guarantee](https://eips.ethereum.org/EIPS/eip-20) a `decimals` function/variable per the EIP, so OZ's `IERC20` won't let us call a `IERC20().decimals()` function due to type checking. We could do a `staticcall` here, but that just seems messy, especially if some ERC20 were to actually NOT implement decimals().

Reply:
TIL :eyes: